site/pages/rest-api/examples/update-operator.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Update Operator
+---
+
+Updates an existing operator with the contents of `operator.xml` (can be a partial XML). This operation requires master operator privileges.
```

Comment:
I think it would be better to have parenthesis statement put in a separate sentence somewhere after this paragraph